docs/tutorials/tagged-releases.md

```diff
@@ -0,0 +1,84 @@
+Check with SRE that the account is set up to handle tagged releases on DockerHub. At the time of writing this, both Springster and TuneMe are set up to handle tagged releases.
+
+### Create the tag
+
+Create the tag locally, on the branch that you want to test in a QA environment. The tag name should be something short and descriptive. For example `api`.
+
+```
+git tag <tag name>
+```
+
+Tell github about the tag by pushing your tag
+
+```
+git push --tags
+```
+
+You can check that that tag has been created by going to the `/releases` section on Github. e.g. `https://github.com/praekelt/molo-tuneme/releases` or more specifically for the tags, `https://github.com/praekelt/molo-tuneme/tags`.
+
+![Tags on Github](../images/github_release_listing.png "Tag listed on Github")
+
+### Check on the Docker Image creation process
+
+If you check Travis, it should automatically start a new build, named after your tagged release.
+
+![Travis runs tests against tagged release](../images/travis_test_tagged_release.png "Travis running tests against the tagged release")
+
+Once all of the tests have passed on Travis, it will bundle everything up and create a Docker Image, containing everything needed to run an instance of your code in production. That image will be tagged with the same tag you used earlier. If everything passes, the Image will then be pushed to the associated account on DockerHub. In this example, it will be `https://hub.docker.com/r/praekeltfoundation/molo-tuneme/tags/`. You should see the tag you created, listed at the top.
+
+![The tag will be listed on Docker Hub](../images/tagged_release_docker_hub.png "The tagged image is listed on Docker Hub")
+
+### Update QA Mission Control
+
+Once you see the tag, Go to QA Mission Control, find the QA instance that you want to test with. Click on the 'edit' link.
+Check the field called `Docker Image`
+
+![The default Docker Image tag is latest](../images/mc_app_image_latest.png "The default Docker Image tag is latest")
+
+Change the listed docker image being used to your tag. In this case it will be `api`.
+
+![Set the Docker image used to your tag](../images/mc_app_image_api.png "Set the Docker image used to your tag")
+
+Finally, hit 'Submit' at the bottom of the page to ensure the changes are made. The site will restart and you should see your changes when it gets back up and running.
+
+## Warnings
+One of the possible gotchas with QA instances is the possiblility of corrupting the DB with migrations. Speak to someone if you're not sure what affects your changes could have.
+
+## Recreating a tagged release
+
+Okay, so you've made some changes and you want to see those new changes in QA.
+
+### Delete the tags
+
+In order to create the new tagged release, you need to delete the tag where it is being used. To do this, first delete the tag on Gihub. Go to the tag listing, which can be found at the URL matching this pattern `https://github.com/praekelt/REPO_NAME/releases/tag/TAG_NAME`.
```

Comment:
Also command line `git push --delete/-d origin <tag name> `. It's the same as deleting a branch: https://stackoverflow.com/a/5480292

Reply:
Cool - added :)